paper/paper.md

```diff
@@ -0,0 +1,281 @@
+  parameters and re-execute when results at a given stage are found not to be satisfactory.
+In this way, CMP3 simplifies the creation of connectomes and makes it a straightforward process
+  even for users not familiar with Nipype and software container technology.
+Nevertheless, it still fulfils the needs of advanced users in charge of analyzing a huge amount of data,
+  offering them the possibility to tune and save all the parameters in configuration files and create a batch
+  job to automatically process all data with the BIDS App.
+
+\textbf{Outputs ready to be consumed in the BIDS ecosystem.}
+CMP3 outputs follow as close as possible the BIDS Derivatives specifications,
+  which facilitates the sharing of the derivatives in the BIDS App ecosystem,
+  and allows the user to easily retrieve any of the files generated by CMP3
+  with tools of the BIDS ecosystem such as pybids [@Yarkoni:2019].
+However, it introduces a new BIDS entity ``atlas-<atlas_label>`` that is used in combination
+  with the ``res-<atlas_scale>`` entity to distinguish imaging and network data derived
+  from different parcellation atlases and scales.
+While the BIDS-Derivatives extension to organize network data is being developed, in which we
+  are actively participating, both structural and functional connectomes generated with CMP3 are
+  saved by default as graph edge lists in ``.tsv`` files, that can be directly analyzed using
+  the \href{https://networkx.org/documentation/stable/tutorial.html}{NetworkX}, a Python library which
+  offers many algorithms and tools to explore graphs and compute local and global network properties.
+Connectivity matrices exported to Matlab as MAT-files and fed to the
+  \href{www.brain-connectivity-toolbox.net}{Brain Connectivity Toolbox}, which is a powerful
+  toolbox containing a large selection of network measures for the characterization of brain
+  connectivity datasets.
+Finally, connectomes can be saved in GraphML format to interface with a lot of general purpose
+  software packages for graph analysis such as \href{www.cytoscape.org}{Cytoscape} or \href{www.gephi.org}{Gephi}.
+Not only this ensures that the connectome files can be opened by
+  the most popular software packages used in this field to perform complex network
+  analyzes, but also this eases the reuse of all outputs in the BIDS ecosystem.
+
+\textbf{Developed with openness, transparency, and good practices in mind.}
+CMP3 is published under the terms of the open source 3-Clause Berkeley Software
+  Distribution (3-Clause BSD) license, which allows unlimited modification, redistribution
+  and commercial use in source and binary forms as long as its copyright notices, and the
+  license's disclaimers of warranty are maintained.
+The source code for CMP3 is hosted at
+  [https://github.com/sebastientourbier/connectomemapper3](https://github.com/sebastientourbier/connectomemapper3),
```

Comment:
@sebastientourbier is the github page https://github.com/connectomicslab/connectomemapper3 or https://github.com/sebastientourbier/connectomemapper3?

Reply:
Good spot